### PR TITLE
Document decision for Control API interaction with Zones

### PR DESCRIPTION
**Pull request by tobru**
This PR documents the decision how the Control API interacts with Zones.

### CODE REVIEW

File: docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+It also takes the xref:references/functional-requirements/portal.adoc[Portal] into consideration, as it plays an important role when interacting with the Control API and its features.
+
+== Proposals
+
+Proposal 1: The Control API connects to zones (Virtual)::
+The Control API provides resources from zones via an aggregated API server.
+It directly connects to all the associated zones API servers and exposes it to the Control API as virtual resources.
+
+Proposal 2: The Control API connects to zones (CRDs)::
+A controller connects to the associated zones API servers and reads or writes remote resources as needed.
```
Comment (simu) [inline docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc:18]:
```suggestion
A controller connects to the associated zones' API servers and reads or writes remote resources as needed.
```

File: docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+== Decision
+
+*Zones connect to the Control API*
+
+== Rationale
+
+Connecting from zones to the Control API has several advantages over the other proposals:
+
+* Only outgoing connections from zones needed, which is usually easier when firewalls are involved
+* Permission model (RBAC) can be easier modelled in the Control API
```
Comment (simu) [inline docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc:35]:
```suggestion
* Permission model (RBAC) is easier to model in the Control API
```

**Review by simu — APPROVED** [review]
LGTM, some wording suggestions inline

File: docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Proposal 1: The Control API connects to zones (Virtual)::
+The Control API provides resources from zones via an aggregated API server.
+It directly connects to all the associated zones API servers and exposes it to the Control API as virtual resources.
+
+Proposal 2: The Control API connects to zones (CRDs)::
+A controller connects to the associated zones API servers and reads or writes remote resources as needed.
+The needed resources are represented as custom resources in the Control API.
+
+Proposal 3: Zones connect to the Control API::
+One ore many agents on the zones connect to the Control API and write and read custom resources.
```
Comment (glrf) [inline docs/modules/ROOT/pages/explanation/decisions/control-api-interaction-with-zones.adoc:22]:
```suggestion
One or many agents on the zones connect to the Control API and write and read custom resources.
```

**Review by glrf — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -15,7 +15,7 @@ else
 	engine_opts ?= --rm --tty --user "$$(id -u)"
 endif
 
-preview_cmd ?= docker run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:3.1.2.3 --antora=docs --style=appuio
+preview_cmd ?= docker run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:3.1.2.3 --antora=docs --style=vshn
```
[the remark this answers is not in the document]
Comment (tobru) [reply Makefile:18]: Jein, it's intended, but might better fit into its own commit, but I didn't fell that this warrants its own PR (its a bug IMHO)

[the remark this answers is not in the document]
Comment (glrf) [reply Makefile:18]: I think it's fine to just do it then

**Pull request merged**